src/components/Button/Button.js

```diff
@@ -1,14 +1,6 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Styled from './Button.style';
 
-class Button extends Component {
-  constructor(props) {
-    super(props);
-  }
-
-  render() {
-    return <Styled.Button {...this.props}>{this.props.children}</Styled.Button>;
-  }
-}
+const Button = (props) => <Styled.Button {...props}>{props.children}</Styled.Button>;
```

Comment:
React Component props 에는 children도 있답니다!
즉 
```suggestion
const Button = (props) => <Styled.Button {...props}></Styled.Button>;
```

이렇게 작성해도 똑같다는 뜻이지요! 👍

Reply:
헉 전혀 몰랐던 부분이네요!!!!!!!! 알려주셔서 감사합니다 😀😀😀